<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpcionesAldeanos</class>
 <widget class="QMainWindow" name="OpcionesAldeanos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>8</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Nombre</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>137</width>
          <height>33</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Civilizacion:</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="plainTextEdit">
        <property name="geometry">
         <rect>
          <x>152</x>
          <y>9</y>
          <width>332</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>90</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Nombre</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>-20</y>
          <width>381</width>
          <height>261</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <zorder>label_11</zorder>
       <zorder>label</zorder>
       <zorder>plainTextEdit</zorder>
       <zorder>pushButton</zorder>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Insertar</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Salud:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Nombre:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Edad:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Genero:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lineEdit_2"/>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="spinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="spinBox_2">
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Insertar</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Inicio</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>Final</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Eliminar</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="1" colspan="2">
         <widget class="QLineEdit" name="lineEdit_3"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Nombre</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Salud</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>Eliminar x Nombre</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="pushButton_5">
          <property name="text">
           <string>Eliminar x Salud</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="spinBox_3">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Eliminar Aldeano</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QPushButton" name="pushButton_6">
          <property name="text">
           <string>Eliminar Edad &gt; 59</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Ordenar</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_7">
          <property name="text">
           <string>Ordenar x Nombre</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_8">
          <property name="text">
           <string>Ordenar x Edad</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="pushButton_9">
          <property name="text">
           <string>Ordenar x Salud</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_10">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Ordenar Aldeanos</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="pushButton_10">
          <property name="text">
           <string>Mostrar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Insertar Barco</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="1">
         <widget class="QLabel" name="label_15">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Insertar Barco</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Combustible:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="pushButton_11">
          <property name="text">
           <string>Agregar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinBox_4">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButton_12">
          <property name="text">
           <string>Mostra Barcos</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Buscar Barco</string>
       </attribute>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>171</x>
          <y>9</y>
          <width>166</width>
          <height>33</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Buscar Barcos</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Id:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_7">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>100</y>
          <width>37</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableWidget" name="tableWidget">
        <property name="geometry">
         <rect>
          <x>93</x>
          <y>48</y>
          <width>391</width>
          <height>126</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_13">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>180</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Mostar</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_14">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>180</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Buscar</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Eliminar Barco</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="2" column="2">
         <widget class="QSpinBox" name="spinBox_9"/>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_18">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Eliminar Barcos</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>Cantidad de Combustible:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>Id del Barco a Eliminar:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QSpinBox" name="spinBox_8"/>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="pushButton_15">
          <property name="text">
           <string>Eliminar ID</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="pushButton_16">
          <property name="text">
           <string>Eliminar Combustible</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>Batalla</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_17">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>209</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Mostar</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_18">
        <property name="geometry">
         <rect>
          <x>419</x>
          <y>209</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Mandar</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="tableWidget_2">
        <property name="geometry">
         <rect>
          <x>22</x>
          <y>42</y>
          <width>391</width>
          <height>161</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>9</y>
          <width>235</width>
          <height>33</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Batalla de Barcos</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>140</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Id:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_11">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>140</y>
          <width>37</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_20">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>209</y>
          <width>321</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Barcos en Batalla</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_21">
        <property name="geometry">
         <rect>
          <x>419</x>
          <y>174</y>
          <width>49</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Velocidad:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_10">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>178</y>
          <width>37</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>14</number>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_9">
       <attribute name="title">
        <string>Recuperar B</string>
       </attribute>
       <widget class="QLabel" name="label_22">
        <property name="geometry">
         <rect>
          <x>138</x>
          <y>9</y>
          <width>346</width>
          <height>33</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Regresar Barco de Mision</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_24">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>185</y>
          <width>62</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Combustible:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_25">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>211</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Armadura:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_6">
        <property name="geometry">
         <rect>
          <x>89</x>
          <y>185</y>
          <width>43</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_12">
        <property name="geometry">
         <rect>
          <x>66</x>
          <y>211</y>
          <width>43</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_21">
        <property name="geometry">
         <rect>
          <x>38</x>
          <y>237</y>
          <width>90</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Barcos en Batalla</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_22">
        <property name="geometry">
         <rect>
          <x>578</x>
          <y>237</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Recuperar</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="tableWidget_3">
        <property name="geometry">
         <rect>
          <x>138</x>
          <y>48</y>
          <width>434</width>
          <height>201</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>686</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
